_data/companies-using-kotlin.yml

```diff
@@ -174,3 +174,22 @@
   url: http://www.99taxis.com
   description: The largest app to request taxis in Brazil! Rewriting our Android apps in Kotlin.
   
+- company: Maxwell Forest Pty Ltd
+  url: https://play.google.com/store/apps/developer?id=Maxwell+Forest+Pty+Ltd
+  description: Safedome (since 1.1.3) and other internal Android apps written in Kotlin.
+  
+- company: Infinum
+  url: https://infinum.co/
+  description: Using Kotlin to build expressive, less verbose Android apps for startups and businesses.
+  
+- company: Hootsuite
+  url: https://hootsuite.com/products/mobile-apps
+  description: Hootsuite's mobile apps schedule, publish, and monitor conversations on Twitter, Facebook, Instagram, YouTube, and more. Using Kotlin in our Android app.
```

Comment:
Hootsuite's using kotlin?

I'm going to have so much fun lambasting the scala guys at [our next meetup](https://www.meetup.com/vancouver-scala/)...